UnitTests/Schemas/IFC4X3_RC4/cubic_100.0_300_1000_1_Meter/Data/cubic_100.0_300_1000_1_Meter.ifc

```diff
@@ -32,7 +32,7 @@ DATA;
 #30 = IFCALIGNMENTSEGMENT('1FNFyHAJeHwuDtwDZHIYIu', #3, $, $, $, #37, #40, #29);
 #34 = IFCRELNESTS('1FNFyHAJeHwuDtwDZHIYIj', $, $, $, #21, (#30));
 #35 = IFCCOMPOSITECURVE((#36), .F.);
-#36 = IFCCURVESEGMENT(.DISCONTINUOUS., #42, IFCPARAMETERVALUE(0.), IFCPARAMETERVALUE(100.), #45);
+#36 = IFCCURVESEGMENT(.DISCONTINUOUS., #42, IFCPARAMETERVALUE(0.), IFCNONNEGATIVELENGTHMEASURE(100.), #45);
```

Comment:
Is there a reason that we had to change this?

Reply:
I thought there was a mistake there, because in other ifc's always come IFCPARAMETERVALUE and than IFCNONNEGATIVELENGTHMEASURE. Is it correct?

Reply:
In this case it doesn't matter, so you can leave it as is or change it: I only have a preference to not change the original file if not necessary.